### ISSUE TITLE
grpcio-reflection does not install with pip

### ISSUE THREAD

**Issue by yingzong** [grpc/grpc#10472]
This is the file in question `grpc/src/python/grpcio_reflection/setup.py`
It's on PyPI: https://pypi.python.org/pypi/grpcio-reflection/1.2.1
But it does not install from PyPI:
```
> sudo pip install --no-cache-dir grpcio-reflection

Collecting grpcio-reflection
  Downloading grpcio-reflection-1.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/tmp/pip-build-xtIHIC/grpcio-reflection/setup.py", line 40, in <module>
        import reflection_commands
    ImportError: No module named reflection_commands
```

However, from the directory itself, `python setup.py install` works.

**Comment by mehrdada** [comment]
Thanks for reporting the issue. It seems for some reason our final artifact packaging script does not include some of the .py files in the directory in the final package.

**Comment by mehrdada** [comment]
This issue is fixed on master by #10482 and subsequent releases should work fine. I am reluctant to ship a new point release to just fix this particular issue (1.3 will arrive shortly) for the package before https://github.com/google/protobuf/issues/2882 is fixed and we can require to a newer version of protobuf, as there is a known issue that affects reflection on some platforms.

**Comment by silkentrance** [comment]
When will the new release (1.3) be available?

**Comment by mehrdada** [comment]
If everything goes well, the plan is to push the 1.3 release on Apr 26.

I should point out the `grpc_reflection` pip package depends on the `protobuf` package, which uses a different code base for its `manylinux` architecture vs. other architectures (the `manylinux` build uses a Python extension written in C++ whereas other architectures use a pure python implementation) and the pure python implementation has a bug that prevents `grpc_reflection` to work correctly, so please keep that in mind. The fix for that is coming in `protobuf` 3.3, but if you use the `manylinux` build of `protobuf` package or compile the C++ extension from source and install it, you should be fine using `grpc_reflection` even now. We have a skipped test in `_reflection_servicer_test.py` that manifests this `protobuf` bug, if unskipped.
